kdl_parser/test/test_kdl_parser.cpp

```diff
@@ -55,36 +52,32 @@ class TestParser : public testing::Test
   {
   }
 
-
   /// Destructor
   ~TestParser()
   {
   }
 };
 
 
-TEST_F(TestParser, test)
-{
-  for (int i=1; i<g_argc-2; i++){
+TEST_F(TestParser, test) {
+  for (int i = 1; i < g_argc - 2; i++) {
     ROS_ERROR("Testing file %s", g_argv[i]);
-    ASSERT_FALSE(treeFromFile(g_argv[i], my_tree));
+    ASSERT_FALSE(kdl_parser::treeFromFile(g_argv[i], my_tree));
   }
 
-  ASSERT_TRUE(treeFromFile(g_argv[g_argc-1], my_tree));
+  ASSERT_TRUE(kdl_parser::treeFromFile(g_argv[g_argc - 1], my_tree));
   ASSERT_EQ(my_tree.getNrOfJoints(), 8);
   ASSERT_EQ(my_tree.getNrOfSegments(), 16);
   ASSERT_TRUE(my_tree.getSegment("dummy_link") == my_tree.getRootSegment());
   ASSERT_EQ(my_tree.getRootSegment()->second.children.size(), (unsigned int)1);
   ASSERT_TRUE(my_tree.getSegment("base_link")->second.parent == my_tree.getRootSegment());
   ASSERT_EQ(my_tree.getSegment("base_link")->second.segment.getInertia().getMass(), 10.0);
```

Comment:
all these should have the liltteral value on the expected (left) side not the actual (right) side